### DISCUSSION TITLE
Length of prompt

### DISCUSSION THREAD

**Discussion by MaazBinMusa** [xtekky/gpt4free#815] [Q&A]
My question is, what is the limit of the length of the input query?

**Comment by bagusindrayana** [comment]
depend on model and provider, remember that we illegally reverse/scraping api from 3rd party services that using open ai model, by default the gpt-3 model is 4097 tokens (maybe 3000+word), but it is possible for 3rd party services to limit it on their servers

**Comment by Charlotte-br560** [comment]
The length limit for input queries varies depending on the specific platform or service you're using. It's best to consult the documentation or guidelines provided by the platform you're working with to determine the exact limit.
